nifi-nar-bundles/nifi-asn1-bundle/nifi-asn1-services/src/main/java/org/apache/nifi/jasn1/JASN1Reader.java

```diff
@@ -247,7 +280,7 @@ private void compileAsnToClass(String... asnFilePaths) {
                 logger.error("ASN.1 file not found [{}]", asn1File, e);
                 parseException = e;
             } catch (TokenStreamException | RecognitionException e) {
-                logger.error("ASN.1 stream parsing failed [{}]", asn1File, e);
+                logger.error("ASN.1 stream parsing failed [{}] due to {}", asn1File, e.toString(), e);
```

Comment:
The general convention of `due to` should be avoided. The stack trace includes the exception message, and that is carried through to Bulletins, so this change should be reverted.